sums.rb

```diff
@@ -0,0 +1,33 @@
+#(4) Create a program sums.rb with two classes.
+
+#(a) a class called Sum1 with an initialize method that takes two parameters and sets the instance variable total to the sum of the two.  Include a line at the top of the class:
+
+#attr_accessor :total
+
+class Sum1
+  attr_accessor :total
+  def initialize(a, b)
+    @total = a + b
+  end
+end
+
+#(b) a class called Sum2 with an initialize method that takes two parameters a and b and sets the instance variable a to the parameter a and the instance variable b to the parameter b. Create a method new_total inside Sum2 that returns the sum of the instance variables a and b.
+
+class Sum2
+  attr_accessor :new_total
+  def initialize(a, b)
+    @a = a
+    @b = b
+    @new_total = @a + @b
+  end
+  #def new_total(a, b)
+  #  @a + @b
+  #end
+end
+
+#(c) In the mainline program, create instances of Sum1 and Sum2, passing 5 and 6 as parameters on the new statement.  Print out the total for Sum1.  Print out the new_total for Sum2.
+
+sum1 = Sum1.new(5, 6)
+puts "The total for Sum1: #{sum1.total}"
+sum2 = Sum2.new(5, 6)
+puts "The new_total for Sum2: #{sum2.new_total}"
```

Comment:
looks good. I think you have an extra attr_accessor on the class Sum2. 
I think you had a question regarding Sum1 and Sum2 and the new_total method. 
If you look at Sum1, you see we have the instance variable **@total.** we also have the line **attr_accessor: :total** line at the beginning of the class. 
this is to show the power of the **attr_accessor** method and how it allows you to read and write a value to a instance variable.
if you look at Sum2 we defined a method new_total that prints out the sum of a and b, so attr_accessor does the work of creating that method for you.

take a look at this StackOverflow answer: 
https://stackoverflow.com/questions/4370960/what-is-attr-accessor-in-ruby?rq=1

also, take a look at the ruby docs here:
https://ruby-doc.org/core-2.4.0/Module.html#method-i-attr_accessor